internal/client.go

```diff
@@ -302,15 +308,20 @@ type (
 		//  - InternalServiceError
 		//  - EntityNotExistError
 		DescribeTaskList(ctx context.Context, tasklist string, tasklistType enums.TaskListType) (*workflowservice.DescribeTaskListResponse, error)
+
+		// CloseConnection closes underlying gRPC connection.
+		CloseConnection() error
 	}
 
 	// ClientOptions are optional parameters for Client creation.
 	ClientOptions struct {
+		HostPort           string
```

Comment:
Document the format and that it is optional. And what happens when it is not specified.